src/PKSim.UI/Binders/ChartDataBinder.cs

```diff
@@ -447,7 +448,7 @@ protected void AssignCurveSeriesList(IEnumerable<Series> seriesList, XYDiagramPa
             var xValue = xAxis.ConvertToDisplayUnit(curveData.XValues[i].X);
             var yValues = yProperty(curveData.YValues[i]);
 
-            double[] yConvertedValues = yValues.Select(y => yAxis.ConvertToDisplayUnit(y)).ToArray();
+            double[] yConvertedValues = yValues.Select(y => yAxis.DisplayValue(y, curveData.YDimension)).ToArray();
```

Comment:
Other key change. Instead of using the yAxis to convert, I am using the YDimension of the curve to convert in the displayh unit of the yAxis.